packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -85,15 +96,16 @@ function optionalContent(copy) {
  * @returns {object} JSX object
  */
 function renderFooter(cta, type) {
+  const Icon = CTALogic.iconSelector(cta.type);
   return (
     cta && (
       <div className={`${prefix}--card__footer`}>
         {type !== 'link' ? (
-          <CTA style="text" {...cta} customClassName={`${prefix}--card__cta`} />
+          <LinkWithIcon type={cta.type} href={cta.href}>
```

Comment:
@RaphaelAmadeu see comment at https://github.com/carbon-design-system/ibm-dotcom-library/pull/2326/files#r425952465. If we're going to only keep this on the `Card` level, you can just pass in the `onClick` behavior here.

cc: @annawen1